README.md

```diff
@@ -21,11 +21,10 @@ Like Gemini, Lagrange has been designed with minimalism in mind. It depends on a
 
 Prebuilt binaries for Windows, macOS and Linux can be found in [Releases][rel]. You can also find [Lagrange on Flathub for Linux](https://flathub.org/apps/details/fi.skyjake.Lagrange).
 
-On macOS you can install and upgrade via a Homebrew tap:
+On macOS you can install and upgrade via a Homebrew:
 
 ```
-$ brew tap skyjake/lagrange
-$ brew install lagrange
+brew install --cask lagrange
```

Comment:
Did you mean to remove the dollar sign? :)
```suggestion
$ brew install --cask lagrange
```

Reply:
That was intentional to match OpenSUSE install instructions 🙃